Source/WebCore/workers/service/ServiceWorkerRegistration.h

```diff
@@ -54,8 +55,9 @@ class ServiceWorkerContainer;
 class WebCoreOpaqueRoot;
 struct CookieStoreGetOptions;
 
-class ServiceWorkerRegistration final : public RefCounted<ServiceWorkerRegistration>, public Supplementable<ServiceWorkerRegistration>, public EventTarget, public ActiveDOMObject, public PushSubscriptionOwner {
+class ServiceWorkerRegistration final : public RefCounted<ServiceWorkerRegistration>, public Supplementable<ServiceWorkerRegistration>, public EventTarget, public ActiveDOMObject, public PushSubscriptionOwner, public CanMakeCheckedPtr<ServiceWorkerRegistration> {
```

Comment:
`ServiceWorkerRegistration` already subclasses `EventTarget`, which means we could use `WeakRef` and avoid subclassing `CanMakeCheckedPtr`